### ISSUE TITLE
Support accessibility

### ISSUE THREAD

**Issue by phuoc-ng** [phuoc-ng/csslayout#127]
Thanks _Andrew Macpherson_ for the feedback:

> It performs poorly on the WCAG principles of perceivable, operable, understandable, and robust. It fails the following WCAG success criteria:
> - 2.1.1 Keyboard
> - 3.3.2 Labels or Instructions
> - 4.1.2 Name, Role, Value
